feg/gateway/services/session_proxy/credit_control/gy/gy_client_test.go

```diff
@@ -138,7 +156,8 @@ func TestGyClient(t *testing.T) {
 	}
 	assert.NoError(t, gyClient.SendCreditControlRequest(&serverConfig, done, ccrTerminate))
 	terminate := gy.GetAnswer(done)
-	assert.Equal(t, len(terminate.Credits), 0)
+	assert.Equal(t, uint32(diam.Success), terminate.ResultCode)
```

Comment:
`assert.Equal(t, diam.Success, int(terminate.ResultCode))` This results in a more readable log when the assertion fails. Actual int value instead of hex